token/core/cache/secondcache/second_chance.go

```diff
@@ -49,13 +49,6 @@ func New(cacheSize int) *secondChanceCache {
 	return &cache
 }
 
-func (cache *secondChanceCache) len() int {
```

Comment:
uiiii we hance the `secondChanceCache` also in FTS. I would suggest that we just use it from there instead of maintaining here a copy.

https://github.com/hyperledger-labs/fabric-smart-client/tree/main/platform/view/services/cache/secondcache

Reply:
Unfortunately it is private and I can't import it (unless I make it public of course).
This code is also copied from `hyperledger/fabric`.

Fortunately, I don't see any usage of this whole file, so most probably I could delete it.

Reply:
ah yes, it seems we got it from Fabric :) I believe we could make it public in FSC if we really needed it here in FTS.

If not needed here .... let's remove